packages/nodes-base/nodes/CrateDb/CrateDb.node.ts

```diff
@@ -373,7 +373,7 @@ export class CrateDb implements INodeType {
 							returning,
 					);
 				}
-				const _updateItems = await db.multi(pgp.helpers.concat(queries));
+				// const _updateItems = await db.multi(pgp.helpers.concat(queries));
```

Comment:
is this unused code?

Reply:
Looks like it, I don't have a cratedb instance running locally but I can't it being used anywhere else.